nvbench/detail/l2flush.cuh

```diff
@@ -31,7 +31,7 @@ struct l2flush
   {
     int dev_id{};
     NVBENCH_CUDA_CALL(cudaGetDevice(&dev_id));
-    NVBENCH_CUDA_CALL(cudaDeviceGetAttribute(&m_l2_size, cudaDevAttrL2CacheSize, dev_id));
+    NVBENCH_CUDA_CALL(cudaDeviceGetAttribute(&dev_id, cudaDevAttrL2CacheSize, dev_id));
```

Comment:
Should this be `&m_l2_size` itself?

Reply:
Oh yeah that is wrong. It should be `&m_l2_size`. But I believe I changed this because if `m_l2_size` is a `size_t`, this doesn't work. I'll find another way to write this.

Reply:
Ok new the updated variant should work :blush: